src/main/js/PipelineEditorLink.jsx

```diff
@@ -15,7 +15,9 @@ class PipelineEditorLink extends React.Component {
         const href = Paths.rest.apiRoot() + '/organizations/' + pipeline.organization + '/pipelines/' + folder + '/';
         pipelineService.fetchPipeline(href, { useCache: true })
         .then(pipeline => {
-            if (pipeline._class === 'io.jenkins.blueocean.blueocean_github_pipeline.GithubOrganizationFolder') {
+            if (pipeline._class === 'io.jenkins.blueocean.blueocean_github_pipeline.GithubOrganizationFolder' ||
```

Comment:
@kzantow because there is no defined capability for editing functionality defined in backend. We need first to add in backend and then consume it here. 

Btw is there requirement to keep editor backward compatible with previous version of blueocean? If we want to keep editor compatible with previous version of blueocean then we will end up keeping the hard coded comparison.